### DISCUSSION TITLE
i made a little QOL change

### DISCUSSION THREAD

**Discussion by Vincevinny** [lllyasviel/Fooocus#877] [Show and tell]
not sure if this is as useful to you as it is to me but i just noticed the program wont be updated for a while,
i made a new start .bat leading to a new entry.py with a new preset and with a new style.
the new start .bat leads to entry_without_update.py which is entry_with_update.py modded to remove the update check (to avoid symlinks\junctions being nuked on start).
the new preset uses juggernautXL_version6Rundiffusion, dpmpp_3m_sde_gpu karras, but you can set it whatever,
this zip can be safely unzipped into an existing install as it doesn't replace any files.
it can also help you understand the programs features that are not yet in the gradio (like setting your defaults or editing styles).

[Fooocus.zip](https://github.com/lllyasviel/Fooocus/files/13262590/Fooocus.zip)

**Comment by AkoZ** [comment]
i just save a copy of my "user_path_config.txt" in fooocus to be sure to get it back !
* the for sure a compare with the new file and some re-copy back...
